.claude/skills/doc-accuracy/scripts/doc_accuracy.py

```
@@ -0,0 +1,1049 @@
+    for i, line in enumerate(lines):
+        match = pattern.match(line)
+        if not match:
+            continue
+
+        kind_token = match.group(2)
+        name = match.group(3)
+
+        if name.startswith("_"):
+            continue
+
+        kind = "class" if kind_token == "class" else "function"
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to: "kind_token"

(S105)